presto-product-tests/src/main/java/com/facebook/presto/tests/AlterTableTests.java

```diff
@@ -32,20 +33,70 @@
     @Test(groups = {ALTER_TABLE, SMOKE})
     public void renameTable()
     {
-        String tableName = "to_be_renamed_table_name";
+        String tableName = "to_be_renamed";
         String renamedTableName = "renamed_table_name";
+
         query(format("DROP TABLE IF EXISTS %s", tableName));
         query(format("DROP TABLE IF EXISTS %s", renamedTableName));
 
-        query(format("CREATE TABLE %s AS SELECT * FROM nation", tableName));
-        assertThat(query(format("ALTER TABLE %s RENAME TO %s", tableName, renamedTableName), UPDATE))
-                .hasRowsCount(1);
+        try {
+            query(format("CREATE TABLE %s AS SELECT * FROM nation", tableName));
+
+            assertThat(query(format("ALTER TABLE %s RENAME TO %s", tableName, renamedTableName), UPDATE))
+                    .hasRowsCount(1);
+
+            assertThat(query(format("SELECT * FROM %s", renamedTableName)))
+                    .hasRowsCount(25);
+
+            // rename back to original name
+            assertThat(query(format("ALTER TABLE %s RENAME TO %s", renamedTableName, tableName), UPDATE))
+                    .hasRowsCount(1);
+        }
+        finally {
+            query(format("DROP TABLE %s", tableName));
```

Comment:
@electrum I agree. Filed a ticket: https://github.com/prestodb/tempto/issues/123. When we address this in Tempto, I will rewrite these tests to use appropriate cleanup mechanism in a separate PR.